src/cgame/cg_main.c

```diff
@@ -823,6 +826,17 @@ void CG_UpdateCvars(void)
 						trap_Cvar_Set("cg_errorDecay", "500");
 					}
 				}
+				else if (cv->vmCvar == &int_cl_extrapolationMargin)
+				{
+					if (int_cl_extrapolationMargin.integer < -2)
+					{
+						trap_Cvar_Set("cl_extrapolationMargin", "-2");
+					}
+					else if (int_cl_extrapolationMargin.integer > 5)
+					{
+						trap_Cvar_Set("cl_extrapolationMargin", "5");
+					}
+				}
```

Comment:
Don't bother adding these to cgame, you don't use it here. Cvar_CheckRange() call after the Cvar_Get in cl_main is better option for range checking an engine cvar for all mods.

Reply:
Makes sense.